Purchases/Purchasing/ProductInfo.swift

```diff
@@ -24,7 +24,7 @@ struct ProductInfo {
     let price: Decimal
     let normalDuration: String?
     let introDuration: String?
-    let introDurationType: PromotionalOffer.IntroDurationType
+    let introDurationType: PromotionalOffer.PaymentMode
```

Comment:
This value isn't actually encoded in `asDictionary`, so the fact that its `rawValue`s are slightly different doesn't have an effect on the API requests.